survos/command-bundle/1.5/config/packages/survos_command.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $container): void {
+    $container->extension('survos_command', [
+        'namespaces' => ['app', 'survos']
+    ]);
+};
+//survos_command:
+//namespaces: [app, project, survos]
```

Comment:
I like PHP config too... though I'm not sure if we ever allow that in the recipes or not...

Reply:
The PHP config seems to work as expected in the recipes, but I switch back to YAML since that's seems to be more common.  Although Tomas Votruba makes great arguments in https://tomasvotruba.com/blog/2020/07/16/10-cool-features-you-get-after-switching-from-yaml-to-php-configs